### PR TITLE
docs: Update README

### PR DESCRIPTION
**Pull request by fghaas**
The site has moved on from an "experimental" to a "beta" state. Update the README accordingly, and add a call to action for anyone interested in contributing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,12 @@
-# Experimental documentation repository
+# Cleura Docs Beta Repository
 
-For the time being, please do not mistake this for anything official
-or supported.
+Here’s where the content of the <https://docs.cleura.cloud>
+documentation web site lives.
+
+This site currently in a Beta state: we do know and understand that
```
Comment (mrtmm) [inline README.md:6]: I think it's missing an 'is' here :)

Comment (fghaas) [reply README.md:6]: Ah-hah! Good catch, eagle-eye :)

Comment (fghaas) [reply README.md]: Fixed.

**Comment by evrardjp** [comment]
Not sure why invalid url - https://docs.openstack.org/octavia/latest/  .

**Comment by evrardjp** [comment]
Ok, force push landed now, will review when the build is passing.

**Comment by fghaas** [comment]
This is so weird. The link checker chokes on <https://docs.openstack.org/octavia/latest/> about half of the time it checks, and the remedy is to just re-run the job.

**Comment by evrardjp** [comment]
For what it's worth, when I was building other static sites with 404 checkers I often got that issue due to rate limiting and other issues. The first step would be to record the HTTP error code, and only then _maybe_ check this with opendev infra.

**Review by evrardjp — APPROVED** [review]

**Pull request merged**